cardano-cli/golden/files/golden/help/shelley_governance_action_create-info.cli

```diff
@@ -0,0 +1,15 @@
+Usage: cardano-cli shelley governance action create-info --epoch EPOCH
+            --drep-cold-verification-key-file FILE
+            [--cost-model-file FILE]
+            --out-file FILE
+
+  Create an info action
+
+Available options:
+  --epoch EPOCH            The epoch number in which the update proposal is
+                           valid.
+  --drep-cold-verification-key-file FILE
+                           Filepath of the drep verification key.
+  --cost-model-file FILE   Filepath of the JSON formatted cost model
```

Comment:
--cost-model-file needs to be replaced with an anchor URL/metadata

Reply:
Added metadata-file/url